File: app/src/main/java/com/art2cat/dev/moonlightnote/controller/settings/AboutAppFragment.java
package com.art2cat.dev.moonlightnote.controller.settings;


import android.os.Bundle;
import com.art2cat.dev.moonlightnote.R;

/**
 * A simple {@link CommonSettingsFragment} subclass.
 */
public class AboutAppFragment extends CommonSettingsFragment {


  public AboutAppFragment() {
    // Required empty public constructor
  }

  @Override
  public String getContent() {
    return getString(R.string.settings_about_app_content);
  }

  @Override
  public CommonSettingsFragment newInstance() {
    AboutAppFragment aboutAppFragment = new AboutAppFragment();
    Bundle args = new Bundle();
    args.putSerializable(SettingsTypeEnum.class.getSimpleName(), SettingsTypeEnum.ABOUT);
    aboutAppFragment.setArguments(args);
    return aboutAppFragment;
  }
}

File: app/src/main/java/com/art2cat/dev/moonlightnote/controller/settings/AboutAppFragment.kt
package com.art2cat.dev.moonlightnote.controller.settings

import android.os.Bundle
import com.art2cat.dev.moonlightnote.R

/**
 * Created by Rorschach
 * on 20/05/2017 9:37 PM.
 */

open class AboutAppFragment : CommonSettingsFragment() {

    override fun getContent(): String {
        return getString(R.string.settings_about_app_content)
    }

    override fun newInstance(): CommonSettingsFragment {
        val aboutAppFragment = AboutAppFragment()
        val args = Bundle()
        args.putInt("type", TYPE_ABOUT_APP)
        aboutAppFragment.arguments = args
        return aboutAppFragment
    }
}
